cockroachcloud/migrations-page.md

```diff
@@ -0,0 +1,159 @@
+- Create a new database that uses the converted schema.
+
+{{site.data.alerts.callout_info}}
+On the **Migrations** page, a *migration* refers to converting a schema for use with CockroachDB and creating a new database that uses the schema. It does not include moving data to the new database. For details on all steps required to complete a database migration, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html).
+{{site.data.alerts.end}}
+
+To view this page, click **Migrations** in the left navigation of the {{ site.data.products.db }} Console. The **Migrations** tab is selected.
+
+## Upload a SQL dump
+
+The upload box for the **Schema Conversion Tool** is displayed at the top of the **Migrations** page.
+
+The **Schema Conversion Tool** expects to analyze a PostgreSQL dump file containing [data definition statements](../{{version_prefix}}sql-statements.html#data-definition-statements) that create a database schema. To generate an appropriate file, run the [`pg_dump` utility](https://www.postgresql.org/docs/current/app-pgdump.html) and specify the `-s` or `--schema-only` options to extract **only the schema** of a PostgreSQL database to a `.sql` file.
+
+{% comment %}
+For an example, see [Migrate a PostgreSQL database](migrate-from-postgres.html#tk).
+{% endcomment %}
+
+{{site.data.alerts.callout_info}}
+The dump file must be less than 4 MB. `INSERT` and `COPY` statements will be ignored in schema conversion.
+{{site.data.alerts.end}}
+
+To begin a database migration:
+
+1. Click the upload box and select a `.sql` file, or drop a `.sql` file directly into the box. 
+1. Wait for the schema to be analyzed. A loading screen is displayed. Depending on the size and complexity of the SQL dump, analyzing the schema can require up to several minutes.
+1. When analysis is complete, review the [**Summary Report**](#summary-report) and edit, add, or remove SQL statements in the [**Statements** list]#statements-list).
+
+## Migrations table
+
+If you have attempted at least one migration, the Migrations table is displayed with the following details:
+
+|     Column     |                                                                                         Description                                                                                          |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Migration Name | The filename of the `.sql` file that was [uploaded](#upload-a-sql-dump).                                                                                                                       |
+| Status         | The status of the migration: `READY FOR REVIEW`, `READY TO FINALIZE`, or `FINALIZED`. Migrations with `READY TO FINALIZE` status can be [finalized](#finalize-the-schema).                    |
```

Comment:
Total nit. To smooth out a little passive, what about:
 
"You can [finalize](#finalize-the-schema) migrations with a `READY TO FINALIZE` status."